app/screens/custom_status/custom_status.tsx

```diff
@@ -128,7 +128,7 @@ function reducer(state: NewStatusType, action: {
             return state;
         }
         case 'text':
-            return {...state, text: action.value};
+            return {...state, text: action.value?.replace('\n', '')};
```

Comment:
should this be a space instead ?
```suggestion
            return {...state, text: action.value?.replace('\n', ' ')};
```

Reply:
That intentional, in my head it made more sense to ignore the input rather than having a space, but I'm 1/5 and easily convinced as I don't know what the right behavior could be